servicepulse/intro-pending-retries.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Pending Retries Message Management in ServicePulse
+summary: Describes how ServicePulse monitors archived failed messages, and allows unarchiving archived failed messages.
+tags:
+ - ServicePulse
+reviewed: 2016-07-28
+---
+
+
+In Versions **1.6.6** and above ServicePulse includes a screen to view and manage failed messages that have been sent for retry but are pending final status.
+
+Pending retries messages can be found by going to the pending retries screen.
+
+![Pending Retries Tab](images/pending-retries.png 'width=500')
+
+
+### Pending Retries Messages
+
+The Pending Retries Messages screen shows failed messages which have been retried but the status of that retry is pending. The status of retried failures is updated when either the message is processed again as either an Audited message (success) or as a failed message.
+
+Failed messages that are retried may stay in the pending state for the following reasons:
+ - The retrying endpoint is not working (e.g. crashed or is scaled-out) and the retried messages is waiting in the queue and has not yet been processed.
+ - The retrying endpoint does not have auditing enabled but has successfully processed the retried message.
+
+To limit the time period of displayed messages select an option from the available predefined time periods, by default pending messages for all time are displayed.
+
+![Period Filter](images/pending-retries-period-selection.png 'width=500')
+
+Results can be filtered by queue name using the search functionality:
+
+![Queue Filter](images/pending-retries-filter-queues.png 'width=500')
+
+Displayed messages display information in the same way as on [Failed Message](intro-failed-messages.md).
+
+ * **Failure Timestamp:** Timestamp when the message have failed.
+ * **Endpoint** Processing endpoint.
+ * **Machine** Machine on which the processing endpoint was operating.
+ * **Redirect** Information if redirect is created for this queue.
+ * **StackTrace:** Displays the full .NET exception stacktrace.
+ * **Headers:** Displays a complete set of message headers.
+ * **Body:** Displays the serialized message body.
+ * **Copy Message Id:** Copies the message unique ID to the clipboard.
+ * **Open in ServiceInsight:** Launches [ServiceInsight](/serviceinsight/), focusing on the message for in-depth analysis of the failure causes. This only works if ServiceInsight is installed on the local machine.
+
+### Message Retry
+
+WARNING: Pending retried messages can be retried however this feature should be used with care. It will create a duplicate of a message to be processed within the system. Person should ensure that this message was not processed or was processed incorrectly before using retry feature.
```

Comment:
>Pending retried messages can be retried however this feature should be used with care. It will create a duplicate of a message to be processed within the system.

How about:

Failed messages that are currently in the pending status can be retried again, however this feature should be used with care. Retrying pending messages can cause the endpoint to reprocess the same message multiple times.